File: src/main/java/com/glitch/knowledge/repositories/StationRepository.java
package com.glitch.knowledge.repositories;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.Query;

import com.glitch.knowledge.model.Station;

public interface StationRepository extends JpaRepository<Station, Integer> {

    @Query(value = "SELECT * FROM station ORDER BY RAND() LIMIT 2", nativeQuery = true)
    Iterable<Station> findTwoRandom();
}

File: src/main/kotlin/com/glitch/knowledge/repository/StationRepository.kt
package com.glitch.knowledge.repository

import com.glitch.knowledge.model.Station
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.Query

interface StationRepository : JpaRepository<Station, Int> {

    @Query(value = "SELECT * FROM station ORDER BY RAND() LIMIT 2", nativeQuery = true)
    fun findTwoRandom(): List<Station>
}